app/js/validateQuestionsFormData.js

```diff
@@ -0,0 +1,180 @@
+let allAnswers = document.querySelectorAll('.answer')
+
+allAnswers.forEach(function(answer, index) {
+    answer.addEventListener('input', () => {
+        event.target.nextElementSibling.innerHTML = "<input type=\"radio\" name=\"answer_radio_button\" value=" + (index + 1) +  "><p>Correct Answer</p>"
+    })
+})
+
+allAnswers.forEach(function (answer) {
+    answer.addEventListener('blur', () => {
+        if (!event.target.value.trim()) {
+            event.target.nextElementSibling.innerHTML = ''
+        }
+    })
+})
+
+document.querySelector('.add_question').addEventListener('submit', function(event) {
+    event.preventDefault()
+
+    let newQuestion = {}
+    let question = document.getElementById('question').value
+    let answer1 = allAnswers[0].value
+    let answer2 = allAnswers[1].value
+    let answer3 = allAnswers[2].value
+    let answer4 = allAnswers[3].value
+    let answer5 = allAnswers[4].value
+    let correctAnswers = document.getElementsByName('answer_radio_button')
+    let questionError = document.querySelector('#question-error')
+    let answer1Error = document.querySelector('#answer1-error')
+    let answer2Error = document.querySelector('#answer2-error')
+    let correctAnswerError = document.querySelector('#correct-answer-error')
+
+    /**
+     * Validates question field and populates newQuestion object with correct question data, otherwise shows error
+     */
+    function validateQuestionText() {
+        if (question.trim().length !==0) {
```

Comment:
`!== 0`

Reply:
ok, add the variable also inside the event listener to make the code more readable